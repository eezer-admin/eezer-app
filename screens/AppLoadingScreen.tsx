import * as React from 'react';
import { View, ActivityIndicator } from 'react-native';

import Styles from '../styles/Styles';

export default function AppLoadingScreen() {
  return (
    <View style={Styles.container}>
      <ActivityIndicator></ActivityIndicator>
    </View>
  );
}
