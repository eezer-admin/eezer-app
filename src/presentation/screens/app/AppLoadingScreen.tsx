import * as React from 'react';
import { ActivityIndicator, View } from 'react-native';

import Styles from '../../../../styles/Styles';

export default function AppLoadingScreen() {
  return (
    <View style={Styles.container}>
      <ActivityIndicator></ActivityIndicator>
    </View>
  );
}
