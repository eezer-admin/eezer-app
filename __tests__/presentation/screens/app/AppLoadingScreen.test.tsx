import { describe, expect, it } from '@jest/globals';
import AppLoadingScreen from '@presentation/screens/app/AppLoadingScreen';
import renderer from 'react-test-renderer';

describe('AppLoadingScreen', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AppLoadingScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
