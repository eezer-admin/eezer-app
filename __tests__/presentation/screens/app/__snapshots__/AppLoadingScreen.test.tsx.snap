// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLoadingScreen renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "alignSelf": "center",
      "flex": 1,
      "justifyContent": "center",
      "justifySelf": "center",
      "paddingHorizontal": 15,
      "width": "100%",
    }
  }
>
  <ActivityIndicator />
</View>
`;
